.github/workflows/tests.yml

```diff
@@ -0,0 +1,65 @@
+name: CI tests
+
+defaults:
+  run:
+    shell: bash
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+        EMULATE_READER: 1
+        # pretend to be cirleci so the scripts dont have to change
+        CIRCLE_WORKING_DIRECTORY: ${{ github.workspace }}
+        GITHUB_PAT: ${{ secrets.GITHUB_PAT }}
+
+    steps:
+      - name: Install
+        run: sudo apt-get update && sudo apt-get install -y luarocks gettext ccache libsdl2-2.0-0
+
+      - name: Shfmt
+        run:  sudo curl -sSL "https://github.com/mvdan/sh/releases/download/v3.4.0/shfmt_v3.4.0_linux_amd64" -o "/usr/local/bin/shfmt" && sudo chmod +x "/usr/local/bin/shfmt"
```

Comment:
And there's the reason for:

https://github.com/koreader/koreader/blob/3dabbd5d73f99e7c23ae752e32a13279a0800f84/.ci/install.sh#L51-L69

Which is annoyingly duplicate with the Docker image in virdevenv. :-)

(Incidentally, it'd surprise me if the GHA shellcheck is quite up to date?)

Reply:
shellcheck 	0.7.0-2build2

Reply:
Yeah, that's old. We want 0.7.2. (Well, actually we want 0.8.0 which just came out going forward but it might catch some new stuff. :-) )

I hadn't updated the shellcheck in .ci/install.sh with the express purpose of switching over to 0.7.2 in the Docker image.